src/panels/config/dashboard/ha-config-updates.ts

```diff
@@ -56,6 +57,26 @@ class HaConfigUpdates extends SubscribeMixin(LitElement) {
       this._entities?.find((entity) => entity.entity_id === entityId)
   );
 
+  private _renderUpdateProgress(entity: UpdateEntity) {
+    if (entity.attributes.update_percentage !== null) {
```

Comment:
```suggestion
    if (entity.attributes.update_percentage != null) {
```
This is safer in case it's `undefined`